parse_hyperparameters.py

```diff
@@ -0,0 +1,35 @@
+import optuna
+import json
+import argparse
+
+
+def value_key(a):
+    if a.value is None:
+        return float('-inf')
+    else:
+        return a.value
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--study-name", help="Study name used during hyperparameter optimization", type=str, default=None)
+parser.add_argument("--storage", help="Database storage path used during hyperparameter optimization", type=str, default=None)
+parser.add_argument("--print-n-best-trials", help="Show final return values for n best trials", type=int, default=0)
+parser.add_argument("--save-n-best-hyperparameters", help="Save the hyperparameters for the n best trials that resulted in the best returns", type=int, default=0)
+args = parser.parse_args()
+
+study = optuna.create_study(study_name=args.study_name, storage=args.storage, load_if_exists=True, direction="maximize")
+values = []
+trials = study.trials
+trials.sort(key=value_key, reverse=True)
+
+for i in trials:
+    if len(values) < args.print_n_best_trials:
+        print(i.value)
+    values.append(i.value)
+
+for i in range(args.save_n_best_hyperparameters):
+    params = trials[i].params
+    text = json.dumps(params)
```

Comment:
the conversion (between named used in the optimization and parameter name) should happen here, no?

Reply:
Why do you think that? I'm confused.

Reply:
If my guess of what you mean is correct, then my response is that this should be fixed elsewhere and not handled here (per https://github.com/DLR-RM/rl-baselines3-zoo/issues/121). All the bugs here for anything I do with rl baselines3 are fixed though.

Reply:
I also think that if conversion is necessary it should happen in the logic that loads the json files

Reply:
> then my response is that this should be fixed elsewhere and not handled here (per #121).

yes this is related to that one. I was wondering where do you handle it currently in your fork?

Reply:
I currently handle it in the logic that loads json files